tests/brevitas_examples/test_quantize_model.py

```diff
@@ -0,0 +1,722 @@
+from copy import deepcopy
+
+import pytest
+import torch
+import torch.nn as nn
+
+from brevitas.nn import QuantConv2d
+from brevitas.nn import QuantLinear
+from brevitas.nn import QuantReLU
+from brevitas.quant_tensor import QuantTensor
+from brevitas_examples.imagenet_classification.ptq.ptq_common import quantize_model
+
```

Comment:
Remaining ToDos, but we can add to it!

Reply:
Save for the missing minifloat tests (see below), I think we can add the other two todos and review/merge this.

Great work!

Reply:
Sure, shall do!

Reply:
I think all of the ToDos (for this PR) are done, subject to whatever changes are desired!

The two tests I added in the latest commit:
- check that the percentiles in stats calibration work as expected, and that the quantization range becomes what we expect it to be.
- check that the MSE calibration method minimizes MSE (perturbed the qparams slightly, and none registered smaller MSE than the original ones).